fmriprep/workflows/fieldmap/se_pair_workflow.py

```diff
@@ -101,41 +97,6 @@ def se_pair_workflow(name='Fieldmap_SEs', settings=None):  # pylint: disable=R09
 
     return workflow
 
-def fieldmap_to_phasediff(name='Fieldmap2Phasediff'):
```

Comment:
Based on this (copied from our email thread): "The sepair_wf workflow was conceived to have the same outputs that any other fieldmap correction workflow would have."

This means that all fieldmap workflows will have similarities (outputs) that make them easy to sub-in/out for each other. It is generally good programming practice, as you know, to factor code so that nothing is repeated. I am most comfortable writing object-oriented style code, so (in my head) the natural way to organize the code would be with an abstract FieldmapWorkflow.py class that defines these outputs, and several fieldmap subclasses that actually implement the necessary workflows. This is just a middle step toward that goal, so I can see how it would not feel justified. The immediate goal here was just to split the fieldmap.py file up into smaller ones, as it was getting monstrously long and was about to get longer.

Does that clarify my thoughts here? Do you have any objections?